notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    torch==2.0.1 transformers==4.36.2 diffusers==0.25.0 "git+https://github.com/huggingface/optimum-intel.git" gradio openvino-nightly
</pre><p>is it necessary to use hard pinned transformers, diffusers and torch versions?</p><p>please also wrap such dependencies in double quotas for correct installation on all platforms</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done